WordPress/src/main/java/org/wordpress/android/ui/mediapicker/MediaPickerFragment.kt

```diff
@@ -28,6 +28,7 @@ import androidx.recyclerview.widget.GridLayoutManager.SpanSizeLookup
 import com.google.android.material.dialog.MaterialAlertDialogBuilder
 import com.google.android.material.snackbar.Snackbar
 import org.wordpress.android.R
+import org.wordpress.android.R.string
```

Comment:
Minor (🔍): This import is not needed and was probably added while you extracted the `navigateEvent(...)` function. Consider removing that and then fixing the code on line `294` with `getString(R.string.cancel)`.

Reply:
Fixed in: 38d215f55bbb5ddbcff4bd105d5eb90c3761e5af

FYI: @Hoossayn you can follow the same pattern and reply like that here, this helps a lot. Then, after me confirming that this commit actually fixes my comment, I can resolve this conversation.

PS: You commit message(s) are still not adhering to the space (` `) standard that I recommended above:
- Instead of: `Refactor:Remove unnecessary explicit string import`
- Do this: `Refactor: Remove unnecessary explicit string import`

Reply:
Noted, thought you meant to stick to one